packages/agreement-process/src/model/domain/agreement-validators.ts

```diff
@@ -262,15 +262,15 @@ export const assertActivableState = (agreement: Agreement): void => {
 export const assertRequesterIsDelegateConsumer = (
   activeConsumerDelegation: Delegation,
   eserviceId: EServiceId,
-  authData: AuthData
+  requesterId: TenantId
```

Comment:
Let's keep `authData` as a parameter:
- It makes the function signature more precise and safer: `TenantId` could be easily switched with the requester or with the delegator at a type-level, they are both tenants, while authData is a specific type without ambiguity
- it's how we do in other validators: https://github.com/pagopa/interop-be-monorepo/pull/1264/files#diff-e4f3ebd287081c5c5c0a081e0f8b52a44687efde74d5cb7e63e6ba1364b903b5R213